include/boost/geometry/extensions/gis/io/shapefile/read.hpp

```diff
@@ -114,8 +114,6 @@ inline void double_endianness_check()
 // TODO: It is not clear that this will work on all machines because some of
 //   them may use mixed endianness (half little-endian, half big-endian) for
 //   doubles or different endianness than the one used for integers.
-//   It is also not clear why did the generic version above compile in the
```

Comment:
Just for the history archive: it's been obsolete as per https://github.com/boostorg/geometry/pull/616#issuecomment-532693484